releasenotes/notes/add-support-for-aiobotocore-1-x-2-x-611d93a3ec362489.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Adds support for aiobotocore 1.x and 2.x
+fixes:
+  - |
+    Prevent aiobotocore patching errors caused by the `ClientResponseContentProxy` import.
```

Comment:
Is this just a byproduct of supporting 2.x? (so we don't have to mention it)

Reply:
Yes, we didn't actually support aiobotocore 1.x or 2.x previously, so this patching issue occurred when an application uses 2.x: https://github.com/DataDog/dd-trace-py/issues/3735 .

Since it's connected to proper support. should I just change this to "feat" ?

Reply:
I just changed this to "feature" to keep it simple. Marking this as resolved!